src/components/pipeline/EditCreatePipeline.js

```diff
@@ -0,0 +1,270 @@
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+  addAlgorithmButton: {
+    fontSize: theme.typography.pxToRem(15),
+    fontWeight: theme.typography.fontWeightRegular,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  content: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    flexDirection: 'column',
+  },
+});
+
+const mockPipeline = {
+  pipelines: [
+    {
+      id: 'pipeline1',
+      name:
+        'Pipel. from hash-users → sanitize-users → suppress-fields → shuffle-fields',
+      algorithms: [
+        { id: 'hash-users', name: 'Hash users' },
+        { id: 'sanitize-users', name: 'Sanitize users' },
+        { id: 'suppress-fields', name: 'Suppress fields' },
+        { id: 'shuffle-fields', name: 'Shuffle fields' },
+      ],
+    },
+    {
+      id: 'pipeline2',
+      name: 'Pipel. from hash-users → k-anonymize-geolocation → shuffle-fields',
+      algorithms: [
+        { id: 'hash-users', name: 'Hash users' },
+        { id: 'k-anonymize-geolocation', name: 'k-Anonymize geolocation' },
+        { id: 'shuffle-fields', name: 'Shuffle fields' },
+      ],
+    },
+  ],
+};
+
+const EditCreatePipeline = (props) => {
+  const { t, classes } = props;
+
+  const [expanded, setExpanded] = useState(false);
+  const [nameP, setNameP] = useState('');
+  const [descriptionP, setDescriptionP] = useState('');
+  const [pipeline, setPipeline] = useState(
+    mockPipeline.pipelines[0].algorithms,
+  );
+
+  const handleChange = (panel) => (_, isExpanded) => {
+    setExpanded(isExpanded ? panel : false);
+  };
+
+  const addPipeline = () => {
+    setPipeline([...pipeline, { id: 'hash-users', name: 'Hash users' }]);
+  };
+
+  const removeAlgorithmFromPipeline = (i) => {
+    const modifiedPipeline = [...pipeline];
+    modifiedPipeline.splice(i, 1);
+    setPipeline(modifiedPipeline);
+  };
+
+  const handleNameOnChange = (event) => {
+    setNameP(event.target.value);
+  };
+
+  const handleDescriptionOnChange = (event) => {
+    setDescriptionP(event.target.value);
+  };
+
+  const PipelineEdit = () => {
+    return (
+      <Container>
+        <h3>{t('Pipeline Schema')}</h3>
+        <div className={classes.pipelineEdit}>
+          {pipeline.map((v, i) => {
+            return (
+              <Accordion
+                expanded={expanded === `panel${i + 1}`}
+                onChange={handleChange(`panel${i + 1}`)}
+              >
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                  id="panel1a-header"
+                >
+                  <ListItemIcon>
+                    <CodeIcon />
+                  </ListItemIcon>
+                  <Typography className={classes.heading}>{v.name}</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <div className={classes.pipelineEdit}>
+                    <Accordion>
```

Comment:
I would actually get rid of these input/output parameters as we won't deal with them for now. Your opinion @hagoptaminian @Painguin ?

Reply:
agreed with @pyphilia . I think in general let's take a minimalist approach to building the UI at this stage... as we don't know how things will evolve once we start building the functionality.

So perhaps... remove everything from here but the algorithm name... i.e. no expandable accordion. we can discuss.